docs/docs/resources/pipeline.yaml

```diff
@@ -0,0 +1,334 @@
+      ${pipeline_name}-output-topic:
+        type: output # required
+        role: topic-role
+        keySchema: key-schema
+        valueSchema: value-schema
+        partitions_count: 1
+        replication_factor: 1
+        configs: 
+          config1: config1
+    models: 
+      model: model
+  # Pipeline prefix that will prefix every component name. If you wish to not 
+  # have any prefix you can specify an empty string.
+  prefix: ${pipeline_name}-
+  # Helm repository configuration, the default value is given here
+  repoConfig:
+    repositoryName: bakdata-streams-bootstrap # required
+    url: https://bakdata.github.io/streams-bootstrap/ # required
+    repo_auth_flags:
+      username: user
+      password: pass
+      ca_file: /home/user/path/to/ca-file
+      insecure_skip_tls_verify: false
+  version: "2.7.0"
+
+# Holds configuration to use as values for the streams bootstrap produce helm
+# chart.
+# More documentation on ProducersApp:
+# https://github.com/bakdata/streams-bootstrap
+- type: producer # required
+  name: producer # required
+  namespace: namespace # required
+  # `app` can contain application-specific settings, hence  the user is free to
+  # add the key-value pairs they need. The fields that are specific to producers
+  # are marked with `# producer-specific`
+  app: # required
+    streams: # required, producer-specific
+      brokers: ${broker} # required
+      schemaRegistryUrl: ${schema_registry_url}
+      outputTopic: output_topic
+      extraOutputTopics: 
+        output_role1: output_topic1
+        output_role2: output_topic2
+    nameOverride: override-with-this-name # kafka-app-specific
+    kkey1:
+      key4: value3
+    key2: value1
+    key3: value2
+  # Topic(s) from which the component will read input
+  from:
+    topics: # required
+      ${pipeline_name}-input-topic:
+        type: input # required
+        role: topic-role
+  # Topic(s) into which the component will write output
+  to:
+    topics: # required
+      ${pipeline_name}-output-topic:
+        type: output # required
+        role: topic-role
+        keySchema: key-schema
+        valueSchema: value-schema
+        partitions_count: 1
+        replication_factor: 1
+        configs: 
+          config1: config1
+    models: 
+      model: model
+  # Pipeline prefix that will prefix every component name. If you wish to not 
+  # have any prefix you can specify an empty string.
+  prefix: ${pipeline_name}-
+  # Helm repository configuration, the default value is given here
+  repoConfig:
+    repositoryName: bakdata-streams-bootstrap # required
+    url: https://bakdata.github.io/streams-bootstrap/ # required
+    repo_auth_flags:
+      username: user
+      password: pass
+      ca_file: /home/user/path/to/ca-file
+      insecure_skip_tls_verify: false
+  version: "2.7.0"
+
+# Kafka source connector
+- type: kafka-source-connector
+  name: kafka-source-connector # required
+  namespace: namespace # required
```

Comment:
Please make it clear that this is the namespace where clean up jobs for the connector will run

Reply:
noted